src/python/module/smartredis/client.py

```diff
@@ -533,8 +533,8 @@ def run_script_multigpu(
         """
         typecheck(name, "name", str)
         typecheck(fn_name, "fn_name", str)
-        typecheck(inputs, "inputs", list)
-        typecheck(outputs, "outputs", list)
+        if not (isinstance(inputs, str) or isinstance(inputs, list)):
+            raise TypeError('Expected str')
```

Comment:
There is a way to pass the combination type into typecheck() that would be better here

Reply:
ah I wasnt able to find that bc I wasnt looking it up right thank u for catching that